stories/3-avatar-images.md

```diff
@@ -12,14 +12,11 @@
 
 ## Existing Behavior
 
-Member records are pulled from an API (https://randomuser.me).
-
-Story #2 rendered the member names and UnreadMessageTag elements in `MemberDataContainer`.
+The member records pulled from the API already include picture data. Check the network response to determine the shape of the response body.
```

Comment:
Do we want to guide the candidates approach with "Check the network response"?

Reply:
This guidance has always existed, just moved it up from the feature requirements, which I do think makes it more clear. I feel fine keeping for now